app/database/operator/server_data_operator/handlers/posts_in_channel.ts

```diff
@@ -202,6 +209,10 @@ const PostsInChannelHandler = (superclass: any) => class extends superclass {
 
         let targetChunk = chunks[0];
         if (targetChunk) {
+            if (targetChunk.earliest <= earliest) {
+                return [];
+            }
+
```

Comment:
Again for my understanding, what use case does this one support?  Does it mean that we don't have the post in the database?